flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/join/temporal/TemporalRowTimeJoinOperator.java

```diff
@@ -150,6 +157,12 @@ public void open() throws Exception {
         joinCondition.setRuntimeContext(getRuntimeContext());
         joinCondition.open(new Configuration());
 
+        tLeftState = getRuntimeContext()
+                .getTemporalListState(new ListStateDescriptor<>(LEFT_STATE_NAME, leftType));
+
+        tRightState = getRuntimeContext()
+                .getTemporalState(new ValueStateDescriptor<>(RIGHT_STATE_NAME, rightType));
+
```

Comment:
I think this change is incomplete, it's unit test in `TemporalRowTimeJoinOperatorTest` is also failing...

Reply:
Oh, it’s not. I didn’t mean to commit that file